src/hdmf/common/__init__.py

```diff
@@ -20,6 +20,34 @@
 # a global type map
 global __TYPE_MAP
 
+@docval({'name': 'config_path', 'type': str, 'doc': 'Path to the configuration file.'},
+        {'name': 'return_map', 'type': bool, 'doc': 'Bool to return type_map', 'default': False},
+        is_method=False)
+def load_termset_config(**kwargs):
+    """
+    This method will either load the default config or the config provided by the path.
+    """
+    config_path, return_map = kwargs['config_path'], kwargs['return_map']
+    __TYPE_MAP.ts_config.load_termset_config(config_path)
+
+    if return_map: # This helps with testing
+        return __TYPE_MAP
```

Comment:
I think this is not needed. You can call `get_type_map` to get a copy of the global `__TYPE_MAP`. It's potentially dangerous to give the user direct access to the global `__TYPE_MAP`. Hence, all the functions below that allow a user to do X to `__TYPE_MAP` instead of just making `__TYPE_MAP` public.

Reply:
So the copy of the type map does not actually have the same configuration loaded. This is why I went with that. Theoretically dangerous, but practically I doubt the users know what to do with it. That is also why the default is false.

Reply:
Having production code that exists solely to make testing easier is generally frowned upon, because the tests should execute the code as if a user was, and they will not / should not be using `return_map=True`. Here, I understand that it is tedious to access the type map, but I think there are other ways to do what you are trying to do (see below)